proto/cosmos/vesting/v1beta1/tx.proto

```diff
@@ -25,6 +25,9 @@ service Msg {
   //
   // Since: cosmos-sdk 0.46
   rpc CreatePeriodicVestingAccount(MsgCreatePeriodicVestingAccount) returns (MsgCreatePeriodicVestingAccountResponse);
+  // DonateAllVestingTokens defines a method that enables donating all vesting 
+  // tokens to community pool
```

Comment:
```suggestion
  // tokens to community pool
  //
  // Since: cosmos-sdk 0.46
```

Reply:
This is a custom msg not found in 0.46 so will not merge your suggestion